multiqc/modules/genomescope2/genomescope2.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python
+
+""" MultiQC module to parse output from genomescope """
+
+from __future__ import print_function
+from collections import OrderedDict
+import logging
+import re, base64
+from multiqc.plots import table
+from multiqc.modules.base_module import BaseMultiqcModule
+
+# Initialise the logger
+log = logging.getLogger(__name__)
+
+
+class MultiqcModule(BaseMultiqcModule):
+    """Genomescope2 module"""
+
+    def __init__(self):
+
+        # Initialise the parent object
+        super(MultiqcModule, self).__init__(
+            name="Genomescope2",
+            anchor="Genomescope2",
+            href="https://github.com/tbenavi1/genomescope2.0",
+            info="Reference-free profiling of polyploid genomes.",
+            doi="https://doi.org/10.1038/s41467-020-14998-3",
+        )
+
+        # Find and load any GENOMESCOPE reports
+        self.summary_data = dict()
+        for f in self.find_log_files("genomescope2/summary"):
+            self.parse_summary_log(f)
+
+        log.info("Found {} summary reports".format(len(self.summary_data)))
+
+        # Write parsed report data to a file
+        self.write_data_file(self.summary_data, "multiqc_genomescope_summary")
+
+        self.add_section(name="Summary", anchor="genomescope-summary", plot=self.summary_table())
+
+        self.plot_figures()
+
+    def parse_summary_log(self, f):
+        self.add_data_source(f, f["s_name"])
+        s_name = f["s_name"].replace("_Summary.txt", "")
+        block = -1
+        i = 0
+        content = []
+        for l in f["f"].splitlines():
+            if l.startswith("property"):
+                block = i
+            if block != -1:
+                content.append(l)
+            if l.startswith("GenomeScope version "):
+                version = l.replace("GenomeScope version ", "")
+            if l.startswith("k = "):
+                k = l.split(" = ")[1]
+            if l.startswith("p = "):
+                p = l.split(" = ")[1]
+            i = i + 1
+        legend = content[0].split()
+        self.summary_data[s_name] = dict()
+        self.legends = []
+        for l in content[1:]:
+            items = re.split(r"\s{2,}", l.strip())
+            this_name = items[0]
+            this_val = items[2]
+            self.summary_data[s_name][this_name] = this_val
+            self.legends.append(this_name)
+
+    def summary_table(self):
+        """Take the parsed stats from the Genomescope report and add some to the
+        General Statistics table at the top of the report"""
+        headers = OrderedDict()
+        for legend in self.legends:
+            headers[legend] = {
+                "title": legend,
+                "description": legend,
```

Comment:
It's a bit difficult to tell without example data, but I suspect that either the titles will be too long, or the descriptions will be too short here 😅 

Also I'm pretty sure that the description takes the title by default if it's not set, so you could just omit it if it's not possible to write longer more helpful descriptions.

Reply:
```suggestion
                "title": legend,
```